src/main/scala/nl/knaw/dans/easy/stage/EasyStageDataset.scala

```diff
@@ -94,6 +99,7 @@ object EasyStageDataset {
 
     def createFileSdo(file: File, parentSDO: String): Try[Unit] = {
       log.debug(s"Creating file SDO for $file")
+      val r = getUserId
```

Comment:
what you mean? `val r` is never used inside this function and neither can it ever be accessed from anywhere outside the function, because it is a *local* value. We probably refer to different lines 😃